### DISCUSSION TITLE
How to actively index new ref or source?

### DISCUSSION THREAD

**Discussion by Financeit-kli** [AltimateAI/vscode-dbt-power-user#63] [Q&A] [answered]
The go to definition function seems not working on new models.

I'm only using it in remote container so I'm not sure if this is the case in local env.

Running dbt list/compile doesn't help

**Comment by mdesmet** [comment, marked as answer]
I don't test with remote containers, but the principle is to point to the correct Python interpreter and that should work as expected.
